presto-hive/src/main/java/com/facebook/presto/hive/BackgroundHiveSplitLoader.java

```diff
@@ -434,7 +434,7 @@ private static boolean shouldUseFileSplitsFromInputFormat(InputFormat<?, ?> inpu
             }
```

Comment:
Please fix the commit message. It is not a revert, at that point of time it was ok to remove this field. There is nothing wrong with that commit so it does not have to be reverted. You just `Add hiveType field to HivePartitionKey`.

Reply:
Done. Changed the commit message.